pkg/volume/fc/disk_manager.go

```diff
@@ -117,5 +119,48 @@ func diskTearDown(manager diskManager, c fcDiskUnmounter, volPath string, mounte
 		}
 	}
 	return nil
+}
+
+// utility to map a device to symbolic link
+func deviceSetUp(manager diskManager, b fcDiskMapper, volPath string) error {
+	globalPDPath := manager.MakeGlobalPDName(*b.fcDisk)
+	if !filepath.IsAbs(globalPDPath) || !filepath.IsAbs(volPath) {
+		return fmt.Errorf("These paths should be absolute: globalPDPath: %s, volPath: %s", globalPDPath, volPath)
+	}
+	devicePath, err := os.Readlink(globalPDPath + "/" + "symlink")
```

Comment:
Who creates this symlink and when? Who deletes it?

Reply:
@jsafrane Under discussion on the spec.